marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/data-services/mapping/getNewDocumentTest.sjs

```diff
@@ -0,0 +1,183 @@
+const test = require("/test/test-helper.xqy");
+const DataHubSingleton = require('/data-hub/5/datahub-singleton.sjs');
+const dataHub = DataHubSingleton.instance();
+const hubTest = require("/test/data-hub-test-helper.sjs");
+
+const jsonUri = '/test/sampleCustomerDoc.json';
+const jsonInput = {
+  envelope:{
+    instance:{
+      CustOrders:{
+        CustomerID:204,
+        Name:{
+          FirstName: 'Jacqueline',
+          LastName: 'Knowles'
+        },
+        Nicknames:{
+          Nickname:[
+            'Jack',
+            'Sparrow'
+          ]
+        }
+      }
+    }
+  }
+};
+
+const xmlUri = '/test/sampleCustomerDoc.xml';
+const xmlInput = `
```

Comment:
I think it'll be cleaner to have one test files for a JSON test, and one test file for an XML test. That's because the XML test is naturally going to be much more interesting. So e.g. getJsonDocumentForTestingTest.sjs and getXmlDocumentForTestingTest.sjs

Reply:
Done.  In the updated or next PR there will be a getDocumentForTesting sub-directory in the suites/data-hub/5/data-services/mapping directory.  At the time of this writing, it includes three files defining tests: nonExistentDocTest.sjs, startWithJson.sjs, and startWithXml.sjs.